ci/tasks/pause-unpause-pipeline.yml

```diff
@@ -0,0 +1,69 @@
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: concourse/concourse-pipeline-resource
+    tag: dev
+  version:
+    digest: sha256:cd899511e06c3027dc8429f4b38ba8ea32a4c0bea044f4711899d5987abf8fdf
+params:
+  ACTION:
+  PIPELINE:
+  FLY_USERNAME:
+  FLY_PASSWORD:
+  MAX_ATTEMPTS: 240 # 1 hour
```

Comment:
Do we need this to be 1 hour? Seems excessive.

Reply:
I've got this to 1 hour because we can be waiting on a database migration in the pipeline (that's the only thing I can think of which might take a long time, everything else should be finished within 5 minutes, but I'd rather take account of that given this will be run automated every day).

Reply:
Oh and it's worth mentioning the migration will take quite a while because it will have to start and stop the database, starting is about 10 minutes and stopping is 10-15 minutes so we know a minimum for a DB migration is going to be 20mins + the migration